lib/msal-node/src/crypto/PkceGenerator.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { PkceCodes } from '@azure/msal-common';
+import { PKCEConstants } from './../utils/NodeConstants';
+import { Base64Encode } from './../encode/Base64Encode';
+const crypto = require('crypto');
+
+/**
+ * https://tools.ietf.org/html/rfc7636#page-8
+ */
+export class PkceGenerator {
+    /**
+     * generates the codeVerfier and the challenge from the codeVerfier
+     * reference: https://tools.ietf.org/html/rfc7636#section-4.1 and https://tools.ietf.org/html/rfc7636#section-4.2
+     */
+    async generatePkceCodes(): Promise<PkceCodes> {
+        const v: string = this.generateCodeVerifier();
+        const c: string = this.generateCodeChallengeFromVerifier(v);
+        return { verifier: v, challenge: c };
```

Comment:
```js
const verifier: string = this.generateCodeVerifier();
const challenge: string = this.generateCodeChallengeFromVerifier(verifier);
return { verifier, challenge };
```

Reply:
I don't think the `: string` parts are needed either

Reply:
Done